src/components/Label/Label.st.css

```diff
@@ -0,0 +1,37 @@
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Label, LabelStyle;
+}
+  
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: secondaryLabel;
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontLight;
+}
+
+.root {
+  -st-extends: Label;
+  -st-states: appearance(string);
```

Comment:
see the comments after, but even if we stick with the `appearance` prop, we should do: `appearance(enum(T1.1,....))`